### PR TITLE
fix permissions

### PR DESCRIPTION
**Pull request by enoch85**
Signed-off-by: enoch85 <github@hanssonit.se>

### CODE REVIEW

**Comment by enoch85** [comment]
Is this what you meant @szaimen?

**Comment by szaimen** [comment]
Yes 👍

File: static/setup_secure_permissions_nextcloud.sh [hunk 1]
```diff
@@ -45,14 +45,16 @@ fi
...
-find "${VMLOGS}"/nextcloud.log -type f -print0 | xargs -0 chmod 0640
 find "${VMLOGS}"/audit.log -type f -print0 | xargs -0 chmod 0640
 find "${NCPATH}"/ -type d -print0 | xargs -0 chmod 0750
+find "${VMLOGS}"/ -type d -print0 | xargs -0 chmod 0750
+find "${VMLOGS}"/nextcloud.log -type f -print0 | xargs -0 chmod 0640
 
 print_text_in_color "$ICyan" "chown Directories"
+chown "${htuser}":"${htgroup}" "${VMLOGS}"/
+chown "${htuser}":"${htgroup}" "${VMLOGS}"/nextcloud.log
+chown "${htuser}":"${htgroup}" "${VMLOGS}"/audit.log
```
Comment (szaimen) [inline static/setup_secure_permissions_nextcloud.sh]: But I am not sure if you need to chown these files explicitly, that are contained in the vmlogs-folder, if you chown the whole folder...

Comment (enoch85) [reply static/setup_secure_permissions_nextcloud.sh]: There are files in there which are root:root which I want to stay as root:root. Hence, I chown the dir itself, and the files needed for it to work.

**Pull request merged**